pyproject.toml

```diff
@@ -22,7 +22,7 @@ packages = [
 python = ">=3.9,<3.10"
 "starknet.py" = "0.18.1"
 cairo-lang = "^0.12"
-starknet-devnet = "^0.5.5"
+starknet-devnet = "^0.6.3"
```

Comment:
Need to upgrade the `starknet-devnet` package to be able to use forked networks at the most recent blocks.